db/migrate/20201127094738_create_placement_need_profiles.rb

```diff
@@ -0,0 +1,23 @@
+class CreatePlacementNeedProfiles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :placement_need_profiles do |t|
+      t.belongs_to :child
+      t.date :placement_date
+      t.boolean :long_term, default: false, null: false
+      t.boolean :short_term, default: false, null: false
+      t.boolean :emergency, default: false, null: false
+      t.boolean :respite, default: false, null: false
+      t.boolean :short_break, default: false, null: false
+      t.boolean :remand, default: false, null: false
+      t.boolean :specialist_theraputic, default: false, null: false
+      t.boolean :parent_and_child, default: false, null: false
+      t.string :addr1
```

Comment:
I've usually seen these prefixed together with `address_` and then `line_1`, `line_2`, `town`, ... So that they are all part of the address. I think because this will repeat in multiple models, we may want to build some abstraction around it later. Maybe even extract it to its own model 🤔

Reply:
Actually thinking about this a bit more, maybe this is not an address. Considering this is only one field, we could go with what's there as `location` and `location_radius`? 🤔  Cause translating one text field into the full structure is not going to work.